### ISSUE TITLE
[Bug] Duration Predictorの学習不良により日本語音声品質が低下

### ISSUE THREAD

**Issue by ayutaz** [ayutaz/piper#157]
# Duration Predictorの学習改善が必要

## 問題
CSS10日本語データセットで学習したモデルの音声品質が低い：
- 無音率が54-65%（正常は30-40%）
- 「こんにちは」が「こ...ん...に...ち...は」と発音される
- 音素間のタイミングがずれる

## 原因
Duration Predictorの学習が不適切：
- `batch_size: 48` が大きすぎた
- Duration loss weightの調整不足
- 特殊トークン（^, $, [, ]）の影響

### 1. 推奨：既存モデルを使用
```python
model = "ja_JP-test-medium.onnx"  # 動作確認済み
```

### 2. 再学習する場合
```bash
python -m piper_train \
  --dataset-dir dataset-css10-ja-prosody \
  --batch-size 32 \     # 48→32
  --max-epochs 2000 \    # 1500→2000  
  --c-dur 0.5           # Duration loss調整
```
※約30-40時間必要

### 3. 現在の回避策
```python
# length_scale=0.3で調整（品質は劣る）
tts = JapaneseTTS(length_scale=0.3)
```

## テスト結果
| モデル | 無音率 | 音声長 | 品質 |
|--------|--------|--------|------|
| 既存 | 37.7% | 2.73秒 | ✅良好 |
| 新規 | 54-65% | 不安定 | ❌不良 |

## 優先度
高（音声品質に直接影響）

## 関連ファイル
- `src/python/piper_train/vits/models.py` - Duration Predictor
- `dataset-css10-ja-prosody/` - 学習データ

**Comment by ayutaz** [comment]
## 詳細情報

### 再現環境
- Dataset: CSS10 Japanese (6,841 files)
- Training: 4x GPU, 1500 epochs
- Framework: PyTorch Lightning 2.4.0
- Branch: feature/cpp-prosody-support

### 定量的評価結果
| 指標 | 既存モデル | 新モデル | 差異 |
|-----|-----------|----------|------|
| 平均無音率 | 37.7% | 54-65% | +16.3-27.3% |
| こんにちは音声長 | 4.18秒 | 1.46秒(scale=0.3) | -65% |
| 音声品質 | 安定 | 不安定 | ❌ |

### 試行した対処法
1. length_scale=0.3での調整 → 部分的改善のみ
2. 設定ファイルの修正 → 効果なし
3. 推論コードの修正 → 根本解決にならず

### 関連ファイル
- Training logs: `output-css10-ja-prosody/lightning_logs/version_0/`
- Generated model: `output-css10-ja-prosody/css10_ja_prosody.onnx`
- Test scripts: `test_existing_model.py`, `test_new_model.py`

### 学習ログ抜粋
```yaml
batch_size: 48
learning_rate: 0.0024 (auto-scaled from 0.0002)
max_epochs: 1500
strategy: ddp_find_unused_parameters_true
```

この問題の解決は日本語TTSの品質向上に必須です。
